CxScaResolver/azure-pipelines.yml

```diff
@@ -0,0 +1,30 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+trigger:
+  - main
+  
+pool:
+  vmImage: ubuntu-latest
+  
+steps:
+  - task: CmdLine@2
+    displayName: Sca Resolver
+    inputs:
+      script: >
+        wget https://sca-downloads.s3.amazonaws.com/cli/latest/ScaResolver-linux64.tar.gz
+  
+        tar -xzvf ScaResolver-linux64.tar.gz
+  
+        rm -rf ScaResolver-linux64.tar.gz 
+        
+  - task: checkmarx.checkmarx-ast-azure-plugin.cx-scan-task.Checkmarx AST@2
```

Comment:
Since we are using an ubuntu image, is it possible on azure to install ubuntu packages? If yes, we could add an example step on installing needed packages for the resolver, like we are doing on the bitbucket resolver example.

Reply:
Added an additional task step to install the pacakges